### ISSUE TITLE
IdNow 6.4.0 crashes after start

### ISSUE THREAD

**Issue by stramek** [idnow/de.idnow.android#184]
### Description
Hello! I'm trying to upgrade SDK version of IdNow in the application, but after upgrade I encountered an issue which crashes the application. I was able to reproduce it with sample IdNow project.

### Reproduce steps

1) Compile sample project https://github.com/idnow/de.idnow.android-sample with SDK `6.4.0`
2) Provide test transaction token to the `IDnowSDK.setTransactionToken(String)` method
3) Launch application
4) Press `Video-identifizierung starten` or `Photo-Identifizierung starten`
5) SDK crash

### Reproduced on devices

- Nexus 5, Android 6.0.1
- Pixel 6, Android 13

I was testing only those two devices.

### Stacktrace

```
FATAL EXCEPTION: main
	Process: de.idnow, PID: 26304
	java.lang.NoSuchFieldError: No static field photo_ident_image_processed of type I in class Lde/idnow/sdk/R$string; or its superclasses (declaration of 'de.idnow.sdk.R$string' appears in /data/app/~~wlGDnwqYZ70tPH2Qb39JnQ==/de.idnow-7XiKEu-jDOyfX8fiepMRLA==/base.apk!classes2.dex)
		at de.idnow.sdk.util.Util_Strings.<clinit>(Util_Strings.java:769)
		at de.idnow.sdk.Activities.Activities_VideoOverviewCheckActivity_Super.overrideContentView(Activities_VideoOverviewCheckActivity_Super.java:280)
		at de.idnow.sdk.Activities.Activities_VideoOverviewCheckActivity.onCreate(Activities_VideoOverviewCheckActivity.java:190)
		at android.app.Activity.performCreate(Activity.java:8341)
		at android.app.Activity.performCreate(Activity.java:8320)
		at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1417)
		at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3622)
		at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3778)
		at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:101)
		at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:138)
		at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
		at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2303)
		at android.os.Handler.dispatchMessage(Handler.java:106)
		at android.os.Looper.loopOnce(Looper.java:201)
		at android.os.Looper.loop(Looper.java:288)
		at android.app.ActivityThread.main(ActivityThread.java:7884)
		at java.lang.reflect.Method.invoke(Native Method)
		at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
		at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

**Comment by stramek** [comment]
Latest version `6.5.0` is also affected with this crash.

**Comment by onursahindur** [comment]
Are you guys even reading the errors here?
As @stramek pointed out the issue still persist on 6.5.0.

**Comment by stramek** [comment]
Fixed in 6.6.0
